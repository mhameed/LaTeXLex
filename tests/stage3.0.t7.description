author: ehc
date: 09/04/01
description: checking that the text surrounding double dollar is left as we would expect - ie. not pulled up