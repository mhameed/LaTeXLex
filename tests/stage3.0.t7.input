a
\bigskip
$$x^2$$
and text which should not be pulled up
