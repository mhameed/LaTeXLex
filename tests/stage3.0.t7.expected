a


x^2
and text which should not be pulled up 
